src/pages/login-success/LoginSuccess.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { useEffect, useCallback } from "react";
 import { useSearchParams, useNavigate } from "react-router-dom";
 
```

Comment:
Ignore this, it was done in https://github.com/kenttk/marks-side-project/pull/26